4. User Guides/Editing Messages/README.md

```diff
@@ -53,3 +53,11 @@ Various emoji shortcuts exists to display popular text representations as graphi
 Graphical emojis can be enabled/disabled in [Your Account](../managing-your-account/) settings under the **Messages** section.
 
 When graphical emojis are disabled, emoji keywords and shortcuts will be displayed as is  e.g. `:smile:` or `:)`.
+
+## Sending video recordings
+
+You can send video recordings by clicking on the video camera icon on the right of the compose message box, which will display a live camera preview and recording controls.
```

Comment:
Right now this is accurate... But This will soon be changed so that accessing is like:

![image](https://user-images.githubusercontent.com/51996/28979716-76454e06-7910-11e7-8a1b-cbee3804d36d.png)
![image](https://user-images.githubusercontent.com/51996/28979725-7d5df666-7910-11e7-8e0b-5850c3715f5f.png)

Just FYI, so we might want an issue for when that does changes so we update this.